### PR TITLE
searching for exhibitors implemented

### PR DESCRIPTION
**Pull request by Jeppes**
https://github.com/armada-ths/armada-android/projects/1#card-289467

### CODE REVIEW

File: app/src/main/java/se/ths/kth/Aramda/components/exhibitors/master/ExhibitorListAdapter.java [hunk 1]
```diff
@@ -142,6 +142,28 @@ private double matchStrength(List<? extends Filterable> list, List<? extends Fil
         return strength;
     }
 
+    public void search(String searchText, List<Exhibitor> exhibitors) {
+        if (searchText.isEmpty()) {
```
Comment (apals) [inline app/src/main/java/se/ths/kth/Aramda/components/exhibitors/master/ExhibitorListAdapter.java]: maybe trim?

File: app/src/main/java/se/ths/kth/Aramda/components/exhibitors/master/ExhibitorListAdapter.java [hunk 2]
```diff
@@ -142,6 +142,28 @@ private double matchStrength(List<? extends Filterable> list, List<? extends Fil
...
+        searchText = searchText.toLowerCase();
+        for (Exhibitor e : exhibitors) {
+            if (e.getName().toLowerCase().startsWith(searchText)) {
+                matches.add(e);
+            }
+        }
+        for (Exhibitor e : exhibitors) {
+            if (e.getName().toLowerCase().contains(searchText) && !matches.contains(e)) {
+                matches.add(e);
+            }
```
Comment (apals) [inline app/src/main/java/se/ths/kth/Aramda/components/exhibitors/master/ExhibitorListAdapter.java]: why not do both of these in just one for loop

File: app/src/main/java/se/ths/kth/Aramda/components/exhibitors/master/ExhibitorListFragment.java [hunk 3]
```diff
@@ -53,16 +57,15 @@
...
-    }
     
     @Override
     public void onResume() {
         super.onResume();
         presenter.start();
+        String searchText = searchBar.getText().toString();
+        if (!searchText.isEmpty()) {
+            searchForExhibitor(searchText);
+        }
```
Comment (Jeppes) [inline app/src/main/java/se/ths/kth/Aramda/components/exhibitors/master/ExhibitorListFragment.java]: yes

**Comment by apals** [comment]
There's a toast saying "Coming soon!" when clicking the search icon.

**Comment by Jeppes** [comment]
fixed

**Comment by apals** [comment]
lgtm

**Pull request merged**